### PR TITLE
Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-aws-lambda-python.git`](https://github.com/cisagov/skeleton-aws-lambda-python.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

---

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Comment by mcdonnnj** [comment]
It may be worth waiting until the merge of https://github.com/cisagov/skeleton-aws-lambda-python/pull/3 and letting that get pushed here for resolution before merging.

**Review by dav3r — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -1,52 +1,57 @@
...
-# Note: Additional labels are added by the build workflow.
-LABEL org.opencontainers.image.authors="nicholas.mcdonnell@cisa.dhs.gov"
+###
+# github@cisa.dhs.gov is a very generic email distribution, and it is
+# unlikely that anyone on that distribution is familiar with the
+# particulars of your repository.  It is therefore *strongly*
+# suggested that you use an email address here that is specific to the
+# person or group that maintains this repository; for example:
+# LABEL org.opencontainers.image.authors="vm-fusion-dev-group@trio.dhs.gov"
+LABEL org.opencontainers.image.authors="github@cisa.dhs.gov"
```
Comment (jsf9k) [inline Dockerfile:40]: You should update this email address, probably to the commented-out suggestion just above.

**Review by jsf9k — APPROVED** [review]
I have one minor suggestion, but this PR is otherwise approved...with gusto!

File: Dockerfile (hunk 1 above)
Comment (jsf9k) [reply Dockerfile:40]: Ah, the `first-commits` branch is still in limbo.  Nevermind.

**Pull request merged**